site/book-theme/header.hbs

```diff
@@ -110,6 +110,108 @@
   padding: 0.25em 0;
   white-space: nowrap;
 }
+
+.main-nav__item:focus {
```

Comment:
Did we create the hamburger menu from scratch?

It would be good to have a hamburger menu for the opentitan.org site as well. I think having the same header on both "sites" would be good for consistency, too.

Reply:
https://github.com/lowRISC/opentitan/pull/17400#issuecomment-1448127997

Reply:
I pretty much followed a tutorial on how to add a hamburger menu. I'm not sure if it's possible to have it be exactly the same header on both, but I can have a look at adding the hamburger menu in the PR that Harry mentions.